conf/susi/susi_cognition_000.json

```diff
@@ -403,6 +404,268 @@
 			"actions":[	{"type":"answer", "select":"random", "phrases":[
 							"You are welcome, too."
 						]}]
+		},
+		{
+
```

Comment:
@sudheesh001 do you recommend running JSONlint on this? It re-formats the file very well.